tasks/get-git-sha-image-ref-from-release/get-git-sha-image-ref-from-release.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: get-git-sha-image-ref-from-release
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "release, tenant"
+spec:
+  description: >-
+    Tekton task to output the imageRef stored in the Release artifacts that corresponds to the git sha tag.
+  params:
+    - name: release
+      type: string
+      description: The namespaced name of the Release
+  results:
+    - name: imageRef
+      type: string
+      description: The imageRef from the Release.Status.Artifacts that uses the git sha tag
+  steps:
+    - name: get-git-sha-image-ref-from-release
+      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      script: |
+        #!/usr/bin/env bash
+        set -euxo pipefail
+
+        IFS='/' read -r RELEASE_NAMESPACE RELEASE_NAME <<< "$(params.release)"
+
+        GIT_SHA=$(kubectl get release "$RELEASE_NAME" -n "$RELEASE_NAMESPACE" \
+          -o jsonpath='{.metadata.labels.pac\.test\.appstudio\.openshift\.io/sha}')
+        if [ -z "$GIT_SHA" ] ; then
+            echo "Error: git sha label from PaC not found in Release labels"
+            exit 1
+        fi
+
+        IMAGES=$(kubectl get release "$RELEASE_NAME" -n "$RELEASE_NAMESPACE" \
```

Comment:
A bit of a nitpick: instead of fetching the release twice, perhaps we could store the json into a variable, and use that to get the GIT_SHA and IMAGES?

Reply:
good idea, will update and post back once done